authors/apps/authentication/migrations/0001_initial.py

```diff
@@ -22,7 +22,7 @@ class Migration(migrations.Migration):
                 ('is_superuser', models.BooleanField(default=False, help_text='Designates that this user has all permissions without explicitly assigning them.', verbose_name='superuser status')),
                 ('username', models.CharField(db_index=True, max_length=255, unique=True)),
                 ('email', models.EmailField(db_index=True, max_length=254, unique=True)),
-                ('is_active', models.BooleanField(default=False)),
+                ('is_active', models.BooleanField(default=True)),
```

Comment:
It had been made False by Esther

Reply:
@collinewait After a rebase, this was changed to True.